tests/routes/business/authorizedPerson.spec.ts

```diff
@@ -0,0 +1,154 @@
+import { expect } from "chai";
+import sinon from "sinon";
+import { mockReq, mockRes } from "sinon-express-mock";
+import * as db from "../../../src/db";
+
+import * as businessesAPI from "../../../src/routes/business/businesses";
+import { createAuthorizedPerson } from "../../../src/routes/business/authorizedPerson";
+import {
+  AccountType,
+  CustomerType,
+  ProductType,
+} from "../../../src/helpers/types";
+import { createAccount } from "../../../src/routes/accounts";
+import { createPerson } from "../../../src/routes/persons";
+
+describe("createAuthorizedPerson", () => {
+  let res: sinon.SinonSpy;
+  let personId: string;
+  let businessId: string;
+  let accountId: string;
+  let req;
+
+  before(async () => {
+    await db.flushDb();
+    res = mockRes();
+
+    await createPerson(
+      {
+        body: {},
+        headers: {},
+      },
+      res
+    );
+
+    personId = res.send.args[0][0].id;
+
+    await businessesAPI.createBusiness(
+      {
+        body: {
+          name: "Kontist GmbH",
+        },
+        headers: {},
+      },
+      res
+    );
+
+    businessId = res.send.args[1][0].id;
+
+    const account = await createAccount(
```

Comment:
you do not need to create an account since you have stubbed the request
also no need in person creation, because you do not check if person with this id exists

Reply:
fixed by https://github.com/kontist/mock-solaris/pull/210/commits/d1882333bca5095bd402bba7b7e7ccc8cf0bbf6e